packages/ia-zendesk-help-widget/src/styles/zendesk-widget.js

```diff
@@ -0,0 +1,43 @@
+import { css } from 'lit-element';
+
+export default css`
+  .help-widget {
+    padding: 13px 22px;
+    margin: 14px 20px;
+    position: fixed;
+    bottom: 0px;
+    z-index: 999998;
+    transition-duration: 250ms;
+    right: 0px;
+    background: #194880;
```

Comment:
**Suggestion**: create CSS variables for background and text color to allow overriding styles based on context.